tb_plugin/torch_tb_profiler/profiler/data.py

```diff
@@ -56,8 +57,15 @@ def parse(run_dir, worker):
         if not os.path.isfile(trace_path):
             raise FileNotFoundError(trace_path)
 
-        with fopen(trace_path, 'r') as f:
-            trace_json = json.load(f)
+        try:
+            with fopen(trace_path, 'r') as f:
+                trace_json = json.load(f)
+        except:
+            with fopen(trace_path, 'r') as f:
+                trace_json = json.load(f, strict=False)
+            fp = tempfile.NamedTemporaryFile('w+t', suffix='.json', delete=False)
```

Comment:
may have permission issue to access temp dir, better to add a try/catch
if fail, still use original json

Reply:
I checked it, that case would not happen in our scenario: https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_utils.py#L304-L318